wp-content/themes/heideandcook/web/sass/_home.scss

```diff
@@ -0,0 +1,141 @@
+@import "settings";
+
+//header
+.header {
+    height: 108px;
+    width: 100%;
+    margin-top: -32px;
+    background-color: $light-gray;
+}
+
+//sliders
+.index-lightbox {
+    height: 500px;
+    margin-top: 0px;
+    margin-bottom: 65px;
+    border-bottom: 12px solid $orange;
+    position: relative;
+    text-align: center;
+    background-image: url('http://www.marketergizmo.com/wp-content/uploads/2015/03/vince-vaughn-images-1.png');
+    background-size: cover;
+
+    .green {
+        width: 100%;
```

Comment:
bor .green and .orange have the same width and height can you move those into the parent or a single `.green, .red` selector?